.github/workflows/lint.yml

```diff
@@ -88,6 +88,22 @@ jobs:
           level: warning
           flags: --linelength=120 --exclude=java/src/main/native/*.c
           filter: "-runtime/references"
+      - name: Install ninja
+        run: python -m pip install --upgrade ninja
+      - name: Generate compile_commands.json
+        run: |
+          python tools/ci_build/build.py \
+            --cmake_generator "Ninja" \
+            --build_dir build \
+            --update \
+            --cmake_extra_defines CMAKE_EXPORT_COMPILE_COMMANDS=ON
+      - name: Run clang-tidy
+        uses: ZedThree/clang-tidy-review@526cbfb043719639f1ebdeedae0cc1eacd219d8f
+        with:
+          token: ${{ secrets.github_token }}
+          build_dir: "build/Debug/compile_commands.json"
```

Comment:
should it be the compile_commands.json directory, "build/Debug"?
https://github.com/ZedThree/clang-tidy-review#inputs

Reply:
no wonder. thanks!